x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/HttpCertificateCommandTests.java

```diff
@@ -589,7 +590,16 @@ private String formatIpAddress(byte[] addr) throws UnknownHostException {
     private String randomPassword() {
         // We want to assert that this password doesn't end up in any output files, so we need to make sure we
         // don't randomly generate a real word.
-        return randomAlphaOfLength(4) + randomFrom('~', '*', '%', '$', '|') + randomAlphaOfLength(4);
+        return randomFrom(
+            "",
+            randomAlphaOfLength(4) + randomFrom('~', '*', '%', '$', '|') + randomAlphaOfLength(4)
+        );
+    }
+
+    private void runForNonEmptyPattern(String pattern, Runnable runnable) {
+        if ("".equals(pattern) == false) {
+            runnable.run();
+        }
     }
```

Comment:
My _preference_ would have been a simple `if` in the code.

Reply:
Sure I removed it and replace with `if` check in the call sites.